mistral_client.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/civisanalytics/mistral_client'
   spec.license       = 'BSD 3-Clause'
 
-  spec.required_ruby_version = '~> 2.3'
+  spec.required_ruby_version = '>= 2.5'
```

Comment:
Or should I do something like `['>= 2.5', '< 3.1']`?

Reply:
Yeah, minor versions can have incompatible changes, so it probably does make sense to specify what we've tested on